kss/builder/decanter/scss/_content.scss

```diff
@@ -141,3 +141,12 @@
 .kss-modifier__example-footer {
   clear: both;
 }
+
+// -----------------------------------------------------------------------------
+// Descriptions
+// -----------------------------------------------------------------------------
+
+.kss-description,
+.kss-modifier__description {
+  max-width: 1000px;
```

Comment:
How come max width?

Reply:
That's for the description for each item in the styleguide. The components themselves are still fullwidth, but I thought the documentation could have a max-width so it's easier to read.